src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const CALC_COUNT = PRICE_PER_DAY * days;
+  const SAIL_FOR_THREE_MORE = 20;
+  const SAIL_FOR_SEVEN_MORE  = 50;
+  const LONG_PERIOD = 7;
+  const SHORT_PERIOD = 3;
+
+  if (days >= LONG_PERIOD) {
+    return CALC_COUNT - SAIL_FOR_SEVEN_MORE;
+  }
+
+  if (days >= SHORT_PERIOD) {
+    return CALC_COUNT - SAIL_FOR_THREE_MORE;
+  }
+
+  if (days > 0) {
```

Comment:
The check for `days > 0` is redundant since the function does not handle negative values for days, and any positive number of days will have been covered by the previous conditions. You can safely remove this condition.